### PR TITLE
Update storage tests for updated test page

### PR DESCRIPTION
**Pull request by sammacbeth**
**Reviewer:**

## Description:

- Update existing assertions to match new cookie names since https://github.com/duckduckgo/privacy-test-pages/pull/109
- Add new assertions for new tests added in test page changes.

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: integration-test/helpers/harness.js [hunk 1]
```diff
@@ -96,6 +95,11 @@ const setup = async (ops) => {
     return { browser, bgPage, requests, teardown, manifestVersion }
 }
 
+function getManifestVersion() {
```
Comment (kzar) [inline integration-test/helpers/harness.js:98]: Why do we need this `getManifestVersion` function, if the `setup` function already returns the manifest version? If `setup` is returning the wrong manifest version, couldn't we fix that instead?

Comment (sammacbeth) [reply integration-test/helpers/harness.js]: setup is only run in the `beforeAll` step of the tests, but we check the manifest version already in the `describe` function of the tests (so it was always undefined). That means that setup happens after we already determined which tests to run.

**Review by kzar — APPROVED** [review]

**Comment by jonathanKingston** [comment]
This test almost always fails on [line 50](https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/develop/integration-test/background/storage.js#L50)

Example: https://github.com/duckduckgo/duckduckgo-privacy-extension/actions/runs/3513357437/jobs/5924291357 there's lots of this line in the actions tests.

**Comment by sammacbeth** [comment]
Looks like just PRs which _don't_ have this commit. Develop branch looks fine (only failure there was a different test)
Screenshot 2022-11-23 at 14 15 05

**Pull request merged**